cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/ApiController.java

```diff
@@ -13,18 +13,448 @@
  */
 package io.mesosphere.mesos.frameworks.cassandra;
 
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.google.common.base.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.StringWriter;
 
 @Path("/")
 public final class ApiController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ApiController.class);
+
+    private final CassandraCluster cluster;
+
+    public ApiController(CassandraCluster cluster) {
+        this.cluster = cluster;
+    }
+
+    @GET
+    @Path("/")
+    @Produces("text/html")
+    public String indexPage() {
+        return "<a href=\"status\">All nodes</a> <br/>" +
+                "<a href=\"seed-nodes\">List of seed nodes</a> <br/>" +
+                "<a href=\"repair/start\">Start repair</a> <br/>" +
+                "<a href=\"repair/status\">Current repair status</a> <br/>" +
+                "<a href=\"repair/last\">Last repair</a> <br/>" +
+                "<a href=\"repair/abort\">Abort current repair</a> <br/>";
+    }
+
+    @GET
+    @Path("/seed-nodes")
+    @Produces("application/json")
+    public Response seedNodes() {
+        StringWriter sw = new StringWriter();
+        try {
+            CassandraFrameworkProtos.CassandraFrameworkConfiguration configuration = cluster.getConfiguration().get();
+
+            JsonFactory factory = new JsonFactory();
+            JsonGenerator json = factory.createGenerator(sw);
+            json.setPrettyPrinter(new DefaultPrettyPrinter());
+            json.writeStartObject();
+
+            json.writeNumberField("native_port", CassandraCluster.getPortMapping(configuration, CassandraCluster.PORT_NATIVE));
```

Comment:
Please update all json properties to use camel case instead of snake case.  This is nice from the JS side of things in that a UI can then stick with camel case convention when accessing properties.